files/fr/web/api/document/getelementbyid/index.md

```diff
@@ -12,7 +12,7 @@ Si vous avez besoin d'accéder à un élément qui n'a pas d'ID, vous pouvez uti
 ## Syntaxe
 
 ```js
-var element = document.getElementById(id);
+const element = document.getElementById(id);
 ```
```

Comment:
Let's change this to be the [same as the English page](https://github.com/mdn/content/blob/main/files/en-us/web/api/document/getelementbyid/index.md?plain=1#L20-L22) :)

````suggestion
```js-nolint
getElementById(id)
```
````

Reply:
I made the change :)